### PR TITLE
[ZEVMITSD-208, ZEVMITSD-150] View Manufacturer account and their users

### PR DESCRIPTION
**Pull request by dae-ne**
https://informedsolutions.atlassian.net/jira/software/c/projects/ZEVMITSD/boards/1290/backlog?selectedIssue=ZEVMITSD-208

### CODE REVIEW

File: src/core/DfT.ZEV.Core.Application/Manufacturers/Queries/GetManufacturerById/GetManufacturerByIdQueryDtoExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        this GetManufacturerByIdQueryDto dto,
+        IEnumerable<IUserInfo>? users)
+    {
+        if (users is null)
+            return dto;
+        
+        var usersList = users.ToList();
+        var updatedUsers = new List<ManufacturerUsersDto>();
+
+        foreach (var manufacturerUser in dto.Users)
```
Comment (kacpersmal) [inline src/core/DfT.ZEV.Core.Application/Manufacturers/Queries/GetManufacturerById/GetManufacturerByIdQueryDtoExtensions.cs:17]: We might try using array.Join, example:
 var mergedArray = array1
            .Join(array2,
                item1 => item1.Id,
                item2 => item2.Id,
                (item1, item2) => new Item
                {
                    Id = item1.Id,
                    Name = item1.Name,
                    Age = item2.Age
                })
            .ToList();

File: src/services/websites/zev-administration-portal/DfT.ZEV.Administration.Web/frontend/css/main.css [hunk 2]
```diff
@@ -0,0 +1,36 @@
+:root {
```
Comment (kacpersmal) [inline src/services/websites/zev-administration-portal/DfT.ZEV.Administration.Web/frontend/css/main.css:1]: Nice!

**Review by kacpersmal — COMMENTED** [review]
Good job! 🤌🔥

File: src/core/DfT.ZEV.Core.Application/Manufacturers/Queries/GetManufacturerById/GetManufacturerByIdQueryDtoExtensions.cs [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+            .GroupJoin(
+                users,
+                dtoUser => dtoUser.Id.ToString(),
+                identityUser => identityUser.Uid,
+                (dtoUsers, identityUsers) => new { u1 = dtoUsers, u2 = identityUsers })
+            .SelectMany(g => g.u2.DefaultIfEmpty(), (g, identityUsers) => new ManufacturerUsersDto
+            {
+                Id = g.u1.Id,
+                Name = identityUsers?.DisplayName,
+                Email = identityUsers?.Email
```
Comment (cruddasj) [inline src/core/DfT.ZEV.Core.Application/Manufacturers/Queries/GetManufacturerById/GetManufacturerByIdQueryDtoExtensions.cs:24]: I think.... that this also covers https://informedsolutions.atlassian.net/browse/ZEVMITSD-150 (which was to include emails from GCP identity platform users)? If you agree, can you update the PR title to 
[ZEVMITSD-208, ZEVMITSD-150] and it should sync the board 👍

**Pull request merged**